Tool is not working if a path to it has spaces in it

If a path to the distro of the tool has spaces in it, tool is not working

E.g. path: `C:\Program Files\text2confl-0.8.0-dist\text2confl`
Error code: `Error: Could not find or load main class Files\text2confl-0.8.0-dist\text2confl\app\*;C:\Program
Caused by: java.lang.ClassNotFoundException: Files\text2confl-0/8/0-dist\text2confl\app\*;C:\Program`

--- a/cli/scripts/plantuml
+++ b/cli/scripts/plantuml
@@ -5,5 +5,4 @@ SCRIPT=$(readlink -f "$0")
 # Absolute path this script is in, thus /home/user/bin
 SCRIPTPATH=$(dirname "$SCRIPT")
 
-java -jar $SCRIPTPATH/plantuml.jar $@
-
+java -jar "$SCRIPTPATH/plantuml.jar" "$@"

--- a/cli/src/assembly/bin/text2confl.cmd
+++ b/cli/src/assembly/bin/text2confl.cmd
@@ -1,3 +1,3 @@
 @echo off
 
-java -cp %~dp0app\*;%~dp0lib\* com.github.zeldigas.text2confl.cli.MainKt %*
+java -cp "%~dp0app\*;%~dp0lib\*" com.github.zeldigas.text2confl.cli.MainKt %*

